lib/quadquizaminos/bottom.ex

```diff
@@ -14,6 +14,13 @@ defmodule Quadquizaminos.Bottom do
   def collides?(bottom, points) when is_list(points) do
     Enum.any?(points, &collides?(bottom, &1))
   end
+   
+  def vulnerable?(bottom, row) do
```

Comment:
This checks if a bottom in a specific row has a vulnerability block. I would suggest the function be named `has_vulnerability_block?`